testbed/README.md

```diff
@@ -1,8 +1,15 @@
 # LivestateTestbed
 
-To start your Phoenix server:
+To run the testbed application run the following commands from the `live_state/testbed/` directory:
+
+  * Install Livestate testbed elixir dependencies with `mix deps.get`
+  * Install Livestate testbed javascript dependencies with `npm install --prefix assets`
+
+  * Install dependencies for phx-live-state and build  `cd ../phx-live-state/ && npm install && npm run build && cd -`
+  * Install dependencies for use-live-state and build  `cd ../use-live-state/ && npm install && npm run build && cd -`
```

Comment:
Can use `npm foo --prefix directoryName` rather than `cd directoryName && npm foo`

Reply:
added a couple setup scripts so it isn't so ugly